src/ocf_lru.c

```diff
@@ -194,7 +194,7 @@ static void remove_lru_list_nobalance(ocf_cache_t cache,
 	node->hot = false;
 }
 
-static void remove_lru_list(ocf_cache_t cache, struct ocf_lru_list *list,
+void remove_lru_list(ocf_cache_t cache, struct ocf_lru_list *list,
```

Comment:
I would rename this to "ocf_lru_remove_locked" since this is now part of LRU list api.